[2024-12-15T09:47:08.723+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-15T09:47:08.732+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: AF_TEST1.trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2 scheduled__2024-12-14T00:00:00+00:00 [queued]>
[2024-12-15T09:47:08.738+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: AF_TEST1.trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2 scheduled__2024-12-14T00:00:00+00:00 [queued]>
[2024-12-15T09:47:08.738+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-15T09:47:08.746+0000] {taskinstance.py:2889} INFO - Executing <Task(TriggerDagRunOperator): trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2> on 2024-12-14 00:00:00+00:00
[2024-12-15T09:47:08.750+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=279) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-15T09:47:08.750+0000] {standard_task_runner.py:72} INFO - Started process 282 to run task
[2024-12-15T09:47:08.750+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'AF_TEST1', 'trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2', 'scheduled__2024-12-14T00:00:00+00:00', '--job-id', '1796', '--raw', '--subdir', 'DAGS_FOLDER/script/AF_TEST1.py', '--cfg-path', '/tmp/tmpbi5ysgz8']
[2024-12-15T09:47:08.751+0000] {standard_task_runner.py:105} INFO - Job 1796: Subtask trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2
[2024-12-15T09:47:08.776+0000] {task_command.py:467} INFO - Running <TaskInstance: AF_TEST1.trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2 scheduled__2024-12-14T00:00:00+00:00 [running]> on host 6418ebfba5fb
[2024-12-15T09:47:08.814+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test-framework' AIRFLOW_CTX_DAG_ID='AF_TEST1' AIRFLOW_CTX_TASK_ID='trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2' AIRFLOW_CTX_EXECUTION_DATE='2024-12-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-14T00:00:00+00:00'
[2024-12-15T09:47:08.815+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-15T09:47:10.904+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:175 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.904+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:222 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.906+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1151 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.906+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1248 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.906+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1271 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.907+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1406 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.907+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1439 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.907+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1495 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.908+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1683 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.908+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1750 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.908+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2025 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.909+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2110 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.909+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2407 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.909+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2468 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.911+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:3987 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.912+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4096 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.912+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4127 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.912+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4186 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.912+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4225 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.913+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4275 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.914+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5209 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.914+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5265 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.915+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5302 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.915+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5358 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.916+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6437 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.916+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6497 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.917+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6538 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.917+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6588 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.917+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7023 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.918+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7096 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.918+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7267 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.918+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7333 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.919+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7378 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.919+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7440 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.920+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8222 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.920+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8282 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.920+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8322 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.921+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8368 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.922+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:9475 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.922+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:9531 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.924+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10532 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.924+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10588 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.924+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10625 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.924+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10680 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.925+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10923 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.925+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10979 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.926+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11016 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.926+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11072 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.926+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11231 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.927+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11282 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.927+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11420 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.927+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11477 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.928+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12300 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.929+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12383 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.929+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12439 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.929+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12524 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.930+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13019 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.930+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13090 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.930+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13139 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.931+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13200 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.932+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13886 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.932+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13941 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.932+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13977 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.932+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14026 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.933+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14318 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.933+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14388 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.934+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14436 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.934+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14492 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.934+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14744 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.935+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14810 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.936+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15415 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.936+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15482 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.936+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15527 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.937+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15578 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.937+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15897 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.937+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15953 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.938+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16617 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.939+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16674 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.939+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16712 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.939+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16780 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.939+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16932 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.940+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16988 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.940+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17025 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.940+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17090 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.941+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17228 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.941+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17284 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.944+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20234 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.944+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20289 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.945+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20862 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.946+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20921 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.946+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20960 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.946+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21014 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.948+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21316 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.948+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21377 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.949+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21418 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.949+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21478 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.949+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21612 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.950+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21668 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.951+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22155 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.951+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22218 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.951+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22559 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.952+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22626 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.952+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22671 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.952+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22722 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.953+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23061 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.953+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23128 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.953+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23173 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.954+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23224 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.954+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23511 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.954+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23567 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.955+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23968 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.955+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:24018 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.957+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25343 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.957+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25398 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.958+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25434 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.958+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25485 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.959+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:26723 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.960+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:26779 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.960+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27121 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.960+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27177 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.961+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27461 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.961+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27517 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.962+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28053 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.962+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28109 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.963+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28536 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.963+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28592 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.964+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28932 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.964+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28982 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.966+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:30727 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.966+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:30783 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.967+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31296 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.967+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31344 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.968+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31617 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.968+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31673 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.969+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32120 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.969+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32176 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.969+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32339 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.970+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32395 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.970+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32432 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.970+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32487 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.972+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33502 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.972+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33558 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.972+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33869 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.973+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33928 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.973+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33967 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.973+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34021 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.974+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34739 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.974+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34806 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.975+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34852 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.975+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34908 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.977+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:36915 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.977+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:36971 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.978+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37417 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.978+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37473 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.979+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37845 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.979+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37901 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.980+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38208 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.980+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38264 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.980+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38301 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.981+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38351 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.981+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38690 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.981+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38758 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.982+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39141 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.982+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39192 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.983+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39419 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.983+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39487 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.983+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39683 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.984+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39738 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.984+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:40019 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.985+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:40084 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.986+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41138 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.986+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41196 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.987+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41359 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.987+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41415 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.987+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41452 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.987+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41507 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.988+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41875 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.988+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41947 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.992+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42441 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.992+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42497 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.992+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42534 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.992+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42590 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.993+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42627 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.993+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42692 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.993+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43020 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.994+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43079 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.994+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43118 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.994+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43168 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.995+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43514 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.995+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43573 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.996+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43612 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.996+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43666 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.996+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43989 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.997+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44045 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.997+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44386 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.998+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44442 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:10.999+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45003 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.000+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45058 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.000+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45444 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.001+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45499 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.002+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46042 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.002+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46098 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.003+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46702 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.003+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46758 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.004+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47000 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.004+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47050 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.005+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47648 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.005+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47704 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.006+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48113 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.006+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48190 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.007+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48244 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.007+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48321 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.009+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49499 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.009+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49555 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.010+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49932 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.010+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50002 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.010+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50052 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.011+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50107 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.011+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50379 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.011+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50449 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.012+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50499 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.012+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50554 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.012+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50590 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.013+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50660 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.013+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50710 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.013+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50770 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.014+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51046 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.014+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51116 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.014+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51166 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.015+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51226 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.015+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51485 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.015+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51540 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.016+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51787 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.016+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51851 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.016+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51895 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.017+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51958 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.017+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52212 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.018+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52275 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.018+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52463 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.018+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52532 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.019+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53040 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.019+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53111 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.020+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53334 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.020+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53407 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.021+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53961 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.021+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54059 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.022+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54322 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.022+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54398 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.023+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55022 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.024+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55078 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.024+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55310 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.024+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55365 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.026+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56196 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.026+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56248 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.026+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56508 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.027+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56564 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.028+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57087 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.028+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57147 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.028+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57187 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.028+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57237 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.029+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57560 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.029+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57620 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.030+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57660 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.030+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57710 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.032+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59106 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.032+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59162 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.033+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59316 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.033+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59396 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.033+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59452 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.034+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59535 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.034+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59594 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.034+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59677 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.034+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59733 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.035+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59818 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.036+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60639 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.036+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60722 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.037+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60778 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.037+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60863 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.037+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61167 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.038+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61250 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.038+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61306 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.038+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61387 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.039+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61696 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.039+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61752 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.040+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61954 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.040+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61992 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.041+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:62660 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.041+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:62715 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.043+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:63882 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.043+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:63946 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.044+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64582 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.044+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64674 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.045+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64940 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.045+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65014 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.046+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65117 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.046+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65199 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.046+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65385 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.047+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65441 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.047+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65874 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.048+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65950 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.048+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66003 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.048+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66087 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.049+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66686 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.050+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66779 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.050+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66918 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.050+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66964 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.052+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:67735 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.052+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:67791 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.052+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68043 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.053+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68097 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.053+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68532 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.054+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68588 SyntaxWarning: invalid escape sequence '\d'
[2024-12-15T09:47:11.331+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-15T09:47:11.332+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=AF_TEST1, task_id=trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2, run_id=scheduled__2024-12-14T00:00:00+00:00, execution_date=20241214T000000, start_date=20241215T094708, end_date=20241215T094711
[2024-12-15T09:47:11.360+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-15T09:47:11.369+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-15T09:47:11.369+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
